onboard-app-guilherme/app/index.tsx

```diff
@@ -0,0 +1,5 @@
+import { Redirect } from "expo-router";
+
+export default function Index() {
+  return <Redirect href="/login" />;
+}
```

Comment:
Acho que não precisa dessa tela só com o redirect para o login, poderia deixar a users como tela inicial, ou fazer ela ser o index mesmo. 

O melhor seria fazer esse redirect só se o usuário estiver deslogado também, aí se o app for aberto e ele já tiver feito login não precisa fazer de novo
```suggestion
export default function Index() {
  if (!isLogged) {
    return <Redirect href="/login" />;
  }
  
  {...render normal da tela} 
}
```

Reply:
Eu pensei em usar essa tela de redirect para ter a lógica da persistência de login. Fazendo um redirecionamento condicional com base na existência do token no asyncstorage. Daria pra manter ela dessa forma?

Reply:
```
if (!isLogged) {
    return <Redirect href="/login" />;
  }
  return <Redirect href="/users" />;
}
```

Seria isso e o isLogged varia de estado conforme o retorno da função getToken que verifica a existência do mesmo